steamclog/src/main/java/com/steamclock/steamclog/LogLevelPreset.kt

```diff
@@ -7,61 +7,44 @@ package com.steamclock.steamclog
  */
 sealed class LogLevelPreset {
 
-    /// Disk: verbose, system: verbose, remote: none
-    object Firehose: LogLevelPreset()
-
-    /// Disk: none, system: debug, remote: none
-    object Develop: LogLevelPreset()
-
-    /// Disk: verbose, system: none, remote: warn
+    object DebugVerbose: LogLevelPreset()
+    object Debug: LogLevelPreset()
     object ReleaseAdvanced: LogLevelPreset()
-
-    /// Disk: none, system: none, remote: warn
     object Release: LogLevelPreset()
 
     val title: String
         get() = when(this) {
-            is Firehose -> "Firehose"
-            is Develop -> "Develop"
-            is ReleaseAdvanced -> "ReleaseAdvanced"
+            is DebugVerbose -> "DebugVerbose"
+            is Debug -> "Debug"
             is Release -> "Release"
+            is ReleaseAdvanced -> "ReleaseAdvanced"
         }
 
-    val global: LogLevel
```

Comment:
Global is not in the spec, and does not appear to be used, so I removed it for now.